Task1.java

```diff
@@ -0,0 +1,24 @@
+public class Task1 {
+
+    //Створити метод findSymbolOccurrence.
+    //Метод приймає як параметр рядок і символ.
+    //Необхідно обчислити, скільки разів символ зустрічається в
+    //переданому рядку і повернути це числове значення
+
+
+    public static void main(String[] args) {
+
+        System.out.println("Number of occurrences is: " + findSymbolOccurrence("Apple", 'p'));
+   }
+
+    public static int findSymbolOccurrence(String str, char symbol) {

```

Comment:
Would be great to add some null validation for the input string, right? Just to think of it: what will happen if you'll pass null as an argument for parameter 'str'?